tests/test-sources/plugins/by-name/snacks/default.nix

```diff
@@ -0,0 +1,60 @@
+{
+  empty = {
+    plugins.snacks.enable = true;
+  };
```

Comment:
Could you also add:
- an `example` configuration which tests the `settingsExample` values.
- a `default` configuration where all options are explicitly set to their default value.